packages/components/molecules/f-searchbox/CHANGELOG.md

```diff
@@ -34,7 +43,7 @@ v4.0.0-beta.13
 *January 05, 2021*
 
 ### Fixed
-- DK & NO field options - provides component with custom attributes, 
+- DK & NO field options - provides component with custom attributes,
```

Comment:
More of an observation; it looks like the line below should be pulled up onto this one.

Reply:
Updated, and fixed some more entries